Casks/fig.rb

```diff
@@ -0,0 +1,36 @@
+cask "fig" do
+  version "1.0.49,270"
+  sha256 :no_check
+
+  url "https://versions.withfig.com/fig@latest.dmg",
+      verified: "versions.withfig.com"
+  name "fig"
+  desc "Reimagine your terminal"
+  homepage "https://fig.io/"
+
+  livecheck do
+    url "https://versions.withfig.com/appcast.xml"
+    strategy :sparkle
+  end
+
+  depends_on macos: ">= :mojave"
+
+  app "Fig.app"
+
+  uninstall script:
+                        {
+                          executable: "~/.fig/tools/uninstall-script.sh",
```

Comment:
Is the script bundled with the app the same as what is moved to the `.fig` directory.
After install the `.fig` directory isn't available straight away, so the below may work?

```suggestion
                          executable: "/Applications/Fig.app/Contents/Resources/uninstall.sh",
```